data/70-browser.conf

```diff
@@ -1 +1 @@
-BROWSER="invoker -s --type=generic /usr/bin/sailjail -p sailfish-browser.desktop /usr/bin/sailfish-browser"
+BROWSER="/usr/bin/invoker --type=browser,silica-qt5 -A -- /usr/bin/sailfish-browser"
```

Comment:
Why do we need to specify 2 booster types here?

Reply:
We don't. It should be enough like "--type=browser".

Reply:
I mean, either we need to use it everywhere or nowhere. Because, for example, we have also media booster. Are we using fallback there too?

Reply:
We are using the same fallback also on camera which is using media booster.